app/org/sagebionetworks/bridge/services/UserProfileServiceImpl.java

```diff
@@ -22,14 +25,17 @@
     
     private SendMailService sendMailService;
 
+    @Autowired
     public void setAccountDao(AccountDao accountDao) {
         this.accountDao = accountDao;
     }
     
+    @Autowired
     public void setExecutorService(ExecutorService executorService) {
         this.executorService = executorService;
     }
```

Comment:
We don't currently have more than one ExecutorService, but we might. Consider changing this to
@Resource(name = "asyncExecutorService")